src/import-data/ImportDataDestination.ts

```diff
@@ -174,9 +174,8 @@ export const ImportDataDestination = (props: ImportDataDestinationProps): ReactN
     : [];
   const canUseTemplateWorkspace = filteredTemplates.length > 0;
 
-  const importMayTakeTimeMessage = p([
-    'Note that the import process may take some time after you are redirected into your destination workspace.',
-  ]);
+  const importMayTakeTimeMessage =
```

Comment:
The paragraph element introduces vertical spacing that made it difficult to manage the various combinations of elements.